includes/managers/icons.php

```diff
@@ -455,6 +532,34 @@ public static function is_migration_allowed() {
 		return $migration_allowed;
 	}
 
+	/**
+	 * Is an icon migration required in the site content.
+	 *
+	 * @since 3.19.0
+	 * @return bool
+	 */
+	public static function is_migration_required() {
+		static $migration_required = false;
+
+		if ( false === $migration_required ) {
+			$needs_upgrade = self::get_needs_upgrade_option();
+			$latest_migration_version = get_option( 'elementor_' . self::LATEST_MIGRATION_OPTION, null );
+
+			// Check if migration is required based on get_needs_upgrade_option() or version comparison
+			$migration_required = null !== $needs_upgrade || ( $latest_migration_version && version_compare( $latest_migration_version, self::LATEST_MIGRATION_REQUIRED_VERSION, '<' ) );
```

Comment:
Not sure that I understand, we use the latest version number (`latest_migration_version`) as part of the flag?